rust/definitions/src/crypto.rs

```diff
@@ -38,7 +38,7 @@
 //! Signer receives Substrate transactions starting in hexadecimal recording
 //! with `53xxyy` where `xx` corresponds to the encryption type.
 //!
-//! For signable transactions, possible encryptions are:
+//! For signable transactions, possible encryption is:
```

Comment:
this is followed by multiple cases do a reword is necessary here somthing like "possible signature schemes are"